### ISSUE TITLE
axe configuration

### ISSUE THREAD

**Issue by jrpool** [cvs-health/testaro#18]
The `axe` test command should allow specification of Axe-core options, including best practices, experimental tests, and labels such as wcag2a, wcag2aa, wcag21a, wcag21aa, best-practice, wcag***, ACT, and cat.*

**Comment by jrpool** [comment]
The `axe` test has been revised in version 4.11.0.

The test now requires a `detailLevel` argument with a value from 0 to 4. The report discloses total rules inapplicable, total rules passed, and totals of rules and nodes with incomplete and violation results. It applies all `axe-core` tests tagged with `wcag2a` through `wcag2aaa`, `wcag21a` through `wcag21aaa`, `best-practice`, and `experimental`.
